tests/python/shared/fixtures/init.py

```diff
@@ -185,12 +185,26 @@ def delete_compose_files():
         filename.unlink(missing_ok=True)
 
 
-def wait_for_server():
-    for _ in range(30):
-        response = requests.get(get_api_url("users/self"))
-        if response.status_code == HTTPStatus.UNAUTHORIZED:
-            break
-        sleep(5)
+def wait_for_services():
+    for i in range(300):
+        logging.getLogger(__package__).debug(f"waiting for the server to load ... ({i})")
```

Comment:
Probably it is better to get the logger only once.

```
logger = logging.getLogger(__package__)
```

Reply:
Changed